packages/jest-circus/src/eventHandler.ts

```diff
@@ -27,6 +27,7 @@ const eventHandler: Circus.EventHandler = (event, state) => {
       break;
     }
     case 'hook_start': {
+      event.hook.startedAt = Date.now();
```

Comment:
should we use `process.hrtime.bigint()` instead? should be more accurate, although it might not matter

Reply:
or `performance.now()` I guess

Reply:
Every other `state.timings.x = ` in this file is `Date.now()`, so I was copying that.

I don't think we are likely to have anyone actually looking at sub-millisecond timings. In my screenshot at the top of the PR, you can see the empty body test takes 2ms to run.